src/Centreon/Infrastructure/MetaServiceConfiguration/API/Model/MetaServiceConfigurationV21Factory.php

```diff
@@ -0,0 +1,84 @@
+<?php
```

Comment:
I think you could look at : https://github.com/centreon/centreon-autodiscovery/pull/434/files#diff-54be133dcd0f313d03c987c66a05279fe8e38a9011bda065d708dda6688baa4b. This way to do your factory could avoid you to have a create your MetaServiceConfigurationV21.php file, and directly create your class in your factory.